include/fschuetz04/simcpp20/simulation.hpp

```diff
@@ -42,17 +43,17 @@ template <typename Time = double> class simulation {
    * @return New pending value event.
    */
   template <typename Value> value_event<Value, Time> event() {
-    return value_event<Value, Time>{*this};
+	return value_event<Value, Time>{*this};
```

Comment:
I don't want to include the formatting changes.

Reply:
I did not use anything :)
When I remember, I format with what's included in Visual Studio.